### ISSUE TITLE
Incorrect username or password - can’t login

### ISSUE THREAD

**Issue by vincentleclerc** [level3tjg/RedditFilter#20]
Side loaded via SideStore, not jailbroken. Latest release 1.1.5

The error appears even though my credentials are correct.

**Comment by level3tjg** [comment]
I'm unable to reproduce but I've seen someone else mention this in another issue (#15). Could you try logging in again? If that doesn't work, let me know what iOS version you're running and I'll take a look.

**Comment by level3tjg** [comment]
I've been able to reproduce this but only once, still no clue what causes it. I was able to login after closing the app from the app switcher

**Comment by level3tjg** [comment]
Could you try installing the build from the [latest action run](https://github.com/level3tjg/RedditFilter/actions/runs/10230434615#artifacts) and logging in again? I believe this is being caused by some sort of device attestation that's required to login. (Download the zip file from the artifacts section and install the contained IPA)

**Comment by Excessiive** [comment]
I’m still having the same issue even with that zip from the artifacts section. I’ve tried everything. Even the log in with a one time link but that brings up a sorry page not found on Reddit in browser.

I was able to get the app to work last month when I was able to log in to an old version (might have been a different repo) and then installing the newer version remained logged in. I can’t seem to get that to work again so if you have any more ideas for this I’m open to trying anything.

Not jailbroken, latest iOS, sideloading with Signulous.

**Comment by p2kdev** [comment]
Same issue here. Not able to login no matter what I try. I am running iOS18 beta 8

**Comment by NicholasBly** [comment]
ios 18.1 b2 unable to login, tried the qnblackcat 1.1.1 ipa as well as RedditPlus (other Reddit tweak) and cannot log in no matter what, only the original Reddit app works for login. Tried deleting original Reddit app, reinstalling and logging in, nothing works. Don't know what to try next

**Comment by Okabeh** [comment]
Same issue, IOS 18 & iPhone 13 Pro. Sideloaded through AltStore and cannot login despite credentials being correct

**Comment by p2kdev** [comment]
Quick update on this. The issue was caused when signing using eSign. I exported ipa using sideloadly & then installed using esign & login works!

**Comment by kylejohnsonkj** [comment]
I'm having this issue using the latest IPA and using my paid developer account to sign with Sideloadly.

Username and password won't work
Google sign in won't work
Cannot use magic link as deeplink won't work

**Comment by NicholasBly** [comment]
> I'm having this issue using the latest IPA and using my paid developer account to sign with Sideloadly.
> 
> Username and password won't work Google sign in won't work Cannot use magic link as deeplink won't work

I solved the issue with downloading an older ipa from 2023, signing into Reddit on the old IPA, then deleting the app and sideloading the new IPA. However the permanent solution is to sideload and sign with signulous as the issue never comes back. A previous commenter says it’s an issue with eSign.

**Comment by kylejohnsonkj** [comment]
@NicholasBly Got it. I wasn't sure what ESign was and if it was something proprietary to one of the signing services. Maybe Sideloadly uses that?

I did try some old app versions but ran into the same thing unfortunately.

**Comment by level3tjg** [comment]
Finally figured out what was causing this, could someone try out the latest changes and confirm that it's fixed? IPA is available from the [latest action run](https://github.com/level3tjg/RedditFilter/actions/runs/11646552983#artifacts)

**Comment by kylejohnsonkj** [comment]
@level3tjg whatever you did did the trick. Thanks!
